pycozmo/audio.py

```diff
@@ -22,25 +22,25 @@ class AudioManager():
     """
     def __init__(self, conn: ClientConnection):
         self.stream = []
-        self.stop = False
+        self._stop = False
```

Comment:
I was getting an exception because a member variable and a method had the same name.

Reply:
Sorry about this, I have to get used to test better my code.. Thank you for fixing it!